### PR TITLE
List entities by query param (q= parameter and corresponding Scope)

### PR DESCRIPTION
**Pull request by fgalan**
Implements: #967 and #864

### CODE REVIEW

File: doc/manuals/user/filtering.md [hunk 1]
```diff
@@ -136,3 +137,35 @@ operation filter.
...
+      }
+      ...
+
+This scope allows to express filtering conditions such as equality, unequality,
+greater/less than, range or existence.
+
+There isn't any URL parameter correspondence for this filter in NGSI v1. In NGSI v2
+it correspons with the `q` parameter:
+
+    curl localhost:1026/v2/entities?q=temperature<24;humidity==75..90;status=running
```
Comment (kzangeli) [inline doc/manuals/user/filtering.md]: This example should be modified, as it wont work ... (adding quotes fixes the problem)

    curl 'localhost:1026/v2/entities?q=temperature<24;humidity==75..90;status=running'

File: doc/manuals/user/filtering.md [hunk 2]
```diff
@@ -136,3 +137,35 @@ operation filter.
...
+              ]
+          }
+      }
+      ...
+
+This scope allows to express filtering conditions such as equality, unequality,
+greater/less than, range or existence.
+
+There isn't any URL parameter correspondence for this filter in NGSI v1. In NGSI v2
+it correspons with the `q` parameter:
```
Comment (kzangeli) [inline doc/manuals/user/filtering.md]: corresponds to

Comment (fgalan) [inline doc/manuals/user/filtering.md]: Fixed in 60e82db

File: doc/manuals/user/filtering.md (hunk 1 above)
Comment (fgalan) [inline doc/manuals/user/filtering.md]: Fixed in 60e82db

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**